src/commands/deploy.rs

```diff
@@ -79,20 +79,26 @@ pub struct DeployCommand {
     /// for readiness.
     #[clap(long = "readiness-timeout", default_value = "60")]
     pub readiness_timeout_secs: u16,
+
+    // Deploy to a named target.
+    #[clap(
+        name = "depoloyment-id",
+        long = "deployment-id",
+        env = DEPLOYMENT_ID_ENV
+    )]
+    pub deployment_id: Option<String>,
```

Comment:
"Deployment ID" for this is rather unintuitive.
Would you consider something like "environment' instead?

I would associate "deployment ID" with an instance of a deployed application that I am in the process of running `spin deploy` for, rather than the environment I am logged into.